scripts/inactive_study.py

```diff
@@ -64,14 +67,23 @@
                        "for an OpenMC model divided into a number of layers")
 ap = ArgumentParser(description=program_description)
 
-ap.add_argument('-i', dest='script_name', type=str,
-                help='Name of the OpenMC python script to run (without .py extension)')
 ap.add_argument('-n-threads', dest='n_threads', type=int,
                 default=multiprocessing.cpu_count(), help='Number of threads to run Cardinal with')
-ap.add_argument('-input', dest='input_file', type=str,
+ap.add_argument('-i', dest='script_name', type=str, required=True,
+                help='Name of the OpenMC python script to run (without .py extension)')
+ap.add_argument('-input', dest='input_file', type=str, required=True,
```

Comment:
For future PRs, try not to unecessarily add diffs if the behavior isn't any different. i.e. the `-i` arg parser line isn't actually any different, you just moved which line it's on. This can make things harder to review.

Reply:
Ah sorry about that. I was getting an error, and for a second thought I needed to put all the non required arguments at the top (but I think that wasn't the problem and my brain was thinking of c++ and defaults). I can put it back if you want. I forgot to move it back once the script wasn't giving errors.

Reply:
Yes, please move it back so that this reflects new content.

Also, is it possible to set a default for `--method`? Good practice is to avoid the user needing to set options when there can be a sensible default. Will be less typing.

Reply:
I think a default of "none" is preferred - those methods you are adding are not foolproof, and I prefer to err on the side of caution of not giving over-confidence to the user